subsys/bluetooth/host/att.c

```diff
@@ -3746,6 +3749,14 @@ void bt_att_set_tx_meta_data(struct net_buf *buf, bt_gatt_complete_func_t func,
 
 	data->func = func;
 	data->user_data = user_data;
+	data->attr_count = 1;
+}
+
+void bt_att_increment_tx_meta_data_attr_count(struct net_buf *buf, uint16_t attr_count)
+{
+	struct bt_att_tx_meta_data *data = bt_att_tx_meta_data(buf);
+
+	data->attr_count += attr_count;
```

Comment:
Do we need to worry about overflow here?

Reply:
I don't think we should, because the `ATT_MULTIPLE_HANDLE_VALUE_NTF` PDU is limited to 16-bit total length. So even with 1-byte attr values, we shouldn't exceed 16bits here. That's my reasoning, at least.

Reply:
Otherwise it's a simple matter of using `u16_add_overflow` with an ASSERT or something if we want to